cameraview/src/main/java/com/otaliastudios/cameraview/CameraView.java

```diff
@@ -1543,6 +1545,24 @@ public int getVideoBitRate() {
         return mCameraEngine.getVideoBitRate();
     }
 
+    /**
+     * A flag to control the behavior when calling {@link #setPreviewFrameRate(float)}.
+     *
+     * If the value is set to true, {@link #setPreviewFrameRate(float)} will choose the preview
+     * frame range as close to the desired new frame rate as possible. Which mean it may choose a
+     * narrow range around the desired frame rate.
```

Comment:
Can you add just a brief comment explaining that this is risky because the sensor will have less freedom when adapting the exposure to the environment, and it can lead to dark preview?